pkg/timeoutconn/timeoutconn.go

```diff
@@ -5,11 +5,13 @@ import (
 	"time"
 )
 
+// New create a net.Conn with a timeout for every Read or Write operation
```

Comment:
Yes, I added a ``FIXME`` comment on the ``conn`` struct (just below) because the comment was wrong. Anyway I'll update that.